.gitignore

```diff
@@ -1 +1,2 @@
 log
+vendor
```

Comment:
what are you trying to ignore ?

Reply:
This is new to me, but it looks like bundle is creating these now:
```
gemfiles/.bundle
vendor/bundle
```